databuilder/databuilder/loader/file_system_atlas_csv_loader.py

```diff
@@ -0,0 +1,191 @@
+
+        self._delete_created_dir = \
+            conf.get_bool(FsAtlasCSVLoader.SHOULD_DELETE_CREATED_DIR)
+        self._force_create_dir = conf.get_bool(FsAtlasCSVLoader.FORCE_CREATE_DIR)
+        self._create_directory(self._entity_dir)
+        self._create_directory(self._relation_dir)
+
+    def _create_directory(self, path: str) -> None:
+        """
+        Validate directory does not exist, creates it, register deletion of
+        created directory function to Job.closer.
+        :param path:
+        :return:
+        """
+        if os.path.exists(path):
+            if self._force_create_dir:
+                LOGGER.info('Directory exist. Deleting directory %s', path)
+                shutil.rmtree(path)
+            else:
+                raise RuntimeError(f'Directory should not exist: {path}')
+
+        os.makedirs(path)
+
+        def _delete_dir() -> None:
+            if not self._delete_created_dir:
+                LOGGER.warning('Skip Deleting directory %s', path)
+                return
+
+            LOGGER.info('Deleting directory %s', path)
+            shutil.rmtree(path)
+
+        # Directory should be deleted after publish is finished
+        Job.closer.register(_delete_dir)
+
+    def load(self, csv_serializable: AtlasSerializable) -> None:
+        """
+        Writes AtlasSerializable into CSV files.
+        There are multiple CSV files that this method writes.
+        This is because there're not only node and relationship, but also it
+        can also have different entities, and relationships.
+
+        Common pattern for both entities and relations:
+         1. retrieve csv row (a dict where keys represent a header,
+         values represent a row)
+         2. using this dict to get a appropriate csv writer and write to it.
+         3. repeat 1 and 2
+
+        :param csv_serializable:
+        :return:
+        """
+
+        entity = csv_serializable.next_atlas_entity()
+        while entity:
+            entity_dict = atlas_serializer.serialize_entity(entity)
+            key = (self._make_key(entity_dict), entity.typeName)
+            file_suffix = '{}_{}'.format(*key)
+            entity_writer = self._get_writer(entity_dict,
+                                             self._entity_file_mapping,
+                                             key,
+                                             self._entity_dir,
+                                             file_suffix)
+            entity_writer.writerow(entity_dict)
+            entity = csv_serializable.next_atlas_entity()
+
+        relation = csv_serializable.next_atlas_relation()
+        while relation:
+            relation_dict = atlas_serializer.serialize_relationship(relation)
+            key2 = (self._make_key(relation_dict),
+                    relation.entityType1,
+                    relation.entityType2)
+
+            file_suffix = f'{key2[0]}_{key2[1]}_{key2[2]}'
```

Comment:
can you assign meaningful variables instead of key2 (unpack the tuple) so it's clear what is the purpose of each of them?